config/rhoai/prometheus_rule.yaml

```diff
@@ -0,0 +1,111 @@
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: High Accelerator Usage.
+          description: Accelerator usage is above 90% of the cluster queue quota for 1 day. Consider increasing available accelerators
+      - alert: LowAcceleratorUsage
+        expr: sum(kueue_cluster_queue_resource_usage{resource="nvidia.com/gpu"}) by (cluster_queue) / sum(kueue_cluster_queue_nominal_quota{resource="nvidia.com/gpu"}) by (cluster_queue) < 0.2
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: Low Accelerator Usage.
+          description: Accelerator usage is below 20% of the cluster queue quota for 1 day. Consider reducing available accelerators
+      - alert: HighRequestedCPUPendingWorkloads
+        expr: (sum by (namespace, pod) ( kube_pod_container_resource_requests{resource="cpu"} * on (namespace, pod) group_left() 
+          kube_pod_status_phase{phase="Pending"})) / 10 > sum(kueue_cluster_queue_nominal_quota{resource="cpu"}) by (cluster_queue)
+        for: 10m
+        labels:
+          severity: info
+        annotations:
+          summary: Pending workload CPU requests far exceed the available quota in cluster queue {{ $labels.cluster_queue }}
+          description: Requested CPU for pending workloads is 10 times the configured quota in the cluster queue {{ $labels.cluster_queue }}
+      - alert: HighRequestedMemoryPendingWorkloads
+        expr: (sum by (namespace, pod) ( kube_pod_container_resource_requests{resource="memory"} * on (namespace, pod) group_left() 
+          kube_pod_status_phase{phase="Pending"})) / 10 > sum(kueue_cluster_queue_nominal_quota{resource="memory"}) by (cluster_queue)
+        for: 10m
+        labels:
+          severity: info
+        annotations:
+          summary: Pending workload memory requests far exceed the available quota in cluster queue {{ $labels.cluster_queue }}
+          description: Requested memory for pending workloads is 10 times the configured quota in the cluster queue {{ $labels.cluster_queue }}
+      - alert: HighRequestedAcceleratorsPendingWorkloads
+        expr: (sum by (namespace, pod) ( kube_pod_container_resource_requests{resource="nvidia_com_gpu"} * on (namespace, pod) group_left() 
+          kube_pod_status_phase{phase="Pending"})) / 10 > sum(kueue_cluster_queue_nominal_quota{resource="nvidia.com/gpu"}) by (cluster_queue)
+        for: 10m
+        labels:
+          severity: info
+        annotations:
+          summary: Pending workload accelerator requests far exceed the available quota in cluster queue {{ $labels.cluster_queue }}
+          description: Number of requested accelerators for pending workloads is 10 times the configured quota in the cluster queue {{ $labels.cluster_queue }}
+      - alert: PendingWorkloadPods
+        expr: sum_over_time(kube_pod_status_phase{phase="Pending", namespace="project-1"}[3d]) >= 3 * 24 * 60
```

Comment:
is `namespace` here required? It looks like an artifact from manual testing

Reply:
Yes you're right, this was leftover from testing. Will update.